src/test/java/com/facebook/openwifirrm/optimizers/TestUtilsTest.java

```diff
@@ -0,0 +1,59 @@
+package com.facebook.openwifirrm.optimizers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.facebook.openwifirrm.ChannelWidth;
+
+public class TestUtilsTest {
```

Comment:
this is a great test to keep this code stable -- feel free to add link to any public reference URL for the encoding logic

Reply:
There is no public reference URL, but I will add a docstring mentioning to look in the 802.11-2020 standard.